core/server/master/src/test/java/alluxio/master/journal/ufs/UfsJournalLogWriterTest.java

```diff
@@ -367,6 +399,7 @@ private DataOutputStream createMockDataOutputStream(UfsJournalLogWriter writer)
       throws IOException {
     flushOutputStream(writer);
     DataOutputStream badOut = Mockito.mock(DataOutputStream.class);
+    Whitebox.setInternalState(badOut, "written", 1);
```

Comment:
Can you update the javadoc for this method indicating that the stream is not wellformed?